androidapp/app/src/main/java/com/example/lanweather/data/DatabaseWorker.kt

```diff
@@ -30,11 +30,11 @@ class DatabaseWorker(
 
                     database.hourlyForecastDao().insertAll(hourlyWeatherList)
 
-                    // Hourly Forecast
+                    // Daily Forecast
```

Comment:
Was this just something that was overlooked, presumably when building the daily forecast dao off of an existing hourly forecast dao?

Reply:
No I think it was just that the wrong code was entered when creating this file. It wasn't referring to the DailyForecast table.